api/src/main/java/org/openmrs/module/sync/SyncUtil.java

```diff
@@ -1387,4 +1388,8 @@ public static String formatTransactionStatus(Transaction tx) {
 		}
 		return "TX STATUS UNKNOWN";
 	}
+

```

Comment:
Can you create a test for this method? According to the method it calculates the days between two dates. with a test scenarios will be much clear.

Reply:
Will do.